### ISSUE TITLE
In Document Viewer, Category check boxes aren't accessible to AT 

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12139]
Once you select a document on the Labels for check box elements are not properly read for assistive technology users. All check boxes read as Object Object. The labels that appear visually with these check boxes Procedural, Medical, Other Evidence and Case Summary should be read for assistive technology users. (New: 08/27/2019)
